cmd/dvotecli/commands/process.go

```diff
@@ -0,0 +1,142 @@
+package commands
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/types"
+	"go.vocdoni.io/dvote/util"
+	"nhooyr.io/websocket"
+)
+
+var processCmd = &cobra.Command{
+	Use:   "process list|keys|results|finalresults|liveresults",
+	Short: "process subcommands",
+}
+
+var processListCmd = &cobra.Command{
+	Use:   "list [entityId]",
+	Short: "list processes of entity",
+	RunE:  processList,
+}
+
+var processKeysCmd = &cobra.Command{
+	Use:   "keys [processId]",
+	Short: "list keys of processes",
+	RunE:  processKeys,
+}
+
+var processResultsCmd = &cobra.Command{
+	Use:   "results [processId]",
+	Short: "get the results of a process",
+	RunE:  getResults,
+}
+
+func init() {
+	rootCmd.AddCommand(processCmd)
+	processCmd.AddCommand(processListCmd)
+	processCmd.AddCommand(processKeysCmd)
+	processCmd.AddCommand(processResultsCmd)
+}
+
+func processList(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf("you must provide a entity id")
+	}
+
+	cl, err := client.New(host)
+	if err != nil {
+		return err
+	}
+	defer cl.Conn.Close(websocket.StatusNormalClosure, "")
+
+	var req types.MetaRequest
+	req.Method = "getProcessList"
```

Comment:
Needs pagination. If the number of processIDs is equal to 64, then use the field "From=64" on a next call to fetch the remaining processes. Do this until the result is empty or the length is smaller than 64

Reply:
Same for all "list" methods.